spec/defines/windows_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'spec_helper'
+describe 'local_user::windows', :type => :define do
+  let (:title) { 'rnelson0' }
+  let (:params) do
+    {
+      :state    => 'present',
+      :groups   => [],
+      :password => 'Microsoft1',
+    }
+  end
+
+  context 'rdp user, no admin' do
+    it { is_expected.to create_user('rnelson0').with({
+      :comment          => 'rnelson0',
+      :groups           => ['Remote Desktop Users'],
+      :password         => 'Microsoft1'
+    }) }
+  end
+  
+  context 'local user only' do
+    let (:params) do
+      {
+        :state     => 'present',
+        :groups    => [],
+        :password  => 'Microsoft1',
+        :admin     => false,
```

Comment:
If this is the default, you do not need to set `admin` here. If you want to, add it to the context description for clarity.